functions/__sf_section_aws.fish

```diff
@@ -0,0 +1,39 @@
+#
+# Amazon Web Services (AWS)
+#
+# The AWS Command Line Interface (CLI) is a unified tool to manage AWS services.
+# Link: https://aws.amazon.com/cli/
+
+function __sf_section_aws -d "Display the selected aws profile"
+	# ------------------------------------------------------------------------------
+	# Configuration
+	# ------------------------------------------------------------------------------
+
+	__sf_util_set_default SPACEFISH_AWS_SHOW true
+	__sf_util_set_default SPACEFISH_AWS_PREFIX "using "
+	__sf_util_set_default SPACEFISH_AWS_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_AWS_SYMBOL "☁️ "
+	__sf_util_set_default SPACEFISH_AWS_COLOR ff8700
```

Comment:
This hex value is the [hex equivalent](https://upload.wikimedia.org/wikipedia/commons/1/15/Xterm_256color_chart.svg) of the xTerm 256 value [used by spaceship](https://github.com/denysdovhan/spaceship-prompt/blob/6065b47ba42dc11fcfd56be924e9896e6b4e6f94/sections/aws.zsh#L15).